compiler/rustc_hir_analysis/src/hir_ty_lowering/lint.rs

```diff
@@ -11,7 +11,7 @@ use super::HirTyLowerer;
 impl<'tcx> dyn HirTyLowerer<'tcx> + '_ {
     /// Prohibit or lint against *bare* trait object types depending on the edition.
     ///
-    /// *Bare* trait object types are ones that aren't preceeded by the keyword `dyn`.
+    /// *Bare* trait object types are ones that aren't preceded by the keyword `dyn`.
```

Comment:
thanks, I recently introduced that mispelling, I always mess that one up like that for some reason ^^'